src/api.ts

```diff
@@ -208,16 +208,23 @@ export class MirrordExecution {
   env: Map<string, string>;
   patchedPath: string | null;
   envToUnset: undefined | string[];
+  usesOperator?: boolean;
```

Comment:
```suggestion
  ¿usesOperator?: boolean;
```

I think we could use a little bit more spanish in our code.

Reply:
what kind of SeñorScript this is

Reply:
:maracas: